### ISSUE TITLE
tweedie

### ISSUE THREAD

**Issue by chanansh** [catboost/catboost#577]
Problem: 
catboost does not support tweedie loss.
The competitors GBT solutions does:
https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst
https://xgboost.readthedocs.io/en/latest/parameter.html

Please consider to add it as it is extremely useful.

**Comment by annaveronika** [comment]
Thanks a lot for pointing this out! We'll consider adding this on our own, but for now we are open to contributions.

**Comment by annaveronika** [comment]
This is implemented now

**Comment by jackohulk99** [comment]
When will it release?
I got "Tweedie loss is not supported" error at ver 0.22

**Comment by annaveronika** [comment]
It will be out in the next release soon, for now you can build from source.

**Comment by Soledadzyh** [comment]
how can i add tweedie loss's param "variance_power", i tried to add it in the init or the fit method , it just cant work

**Comment by annaveronika** [comment]
> how can i add tweedie loss's param "variance_power", i tried to add it in the init or the fit method , it just cant work

Please read the docs here:
https://catboost.ai/docs/concepts/loss-functions.html

**Comment by Arsa-Nik** [comment]
I am using catboost version of 0.26 on windows and R. 
when using Tweedie Loss, it errors out asking for variance_power parameter so it is not defaulted to 1.5 as lightgbm
when variance_power is specified, it doesn't accept the parameter and tells it is an unknown option.
bottom-line: it seems catboost doesn't work with Tweedie loss
